### ISSUE TITLE
NVDA does not announce capitalisation of letters when selecting text

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2670]
**Reported by elliott94 on 2012-09-12 18:03**
When highlighting text using shift and left/right arrows, it's not currently possible to determine whether the selected letter is capitalised or not; NVDA simply anounces the letter, and doesn't take the capital pitch change setting into consideration.

**Comment by leonardder** [comment]
CC @Elliott94

Closing in favour of #4874, which has more replies. This is likely to get fixed by #4877.
